src/lib/util/options.mli

```diff
@@ -98,9 +95,15 @@ val set_debug_explanations : bool -> unit
 (** Set {!val:debug_fm} *)
 val set_debug_fm : bool -> unit
 
+(** Set {!val:debug_fpa} *)
```

Comment:
The `debug_fpa` value is not exposed by this mli file, thus the doc link will be broken, and the documentation is not very useful to determine what the flag controls.
This is also a problem for other doc comments in this file.

Reply:
Exactly and as I already said that's not the purpose of this PR. The documentation has been broken for ages now and I don't know what most of these options do. I suggest you open a PR and everytime we stumble on one of them we'll take the opportunity to update the PR until all the documentation has been resolved. I already did a big chunk of cleaning and it's not pleasant nor rewarding so would you kindly stop hijacking this PR for what it's not supposed to fix?